test/integration/associations/scope.test.js

```diff
@@ -323,105 +323,110 @@ describe(Support.getTestDialectTeaser('associations'), () => {
-
-              // Flaky test on MySQL8: randomly some values will be 0 sometimes, for
-              // now no solution. Not reproducible at local or cloud with logging enabled
-              expect([
-                postACategories.length,
-                postATags.length,
-                postBCategories.length,
-                postBTags.length,
-                postCCategories.length,
-                postCTags.length
-              ]).to.eql([1, 1, 1, 1, 1, 1]);
-
-              expect([
-                postACategories[0].get('type'),
-                postATags[0].get('type'),
-                postBCategories[0].get('type'),
-                postBTags[0].get('type'),
-                postCCategories[0].get('type'),
-                postCTags[0].get('type')
-              ]).to.eql(['category', 'tag', 'category', 'tag', 'category', 'tag']);
-
-              const [postA, postB, postC] = await Promise.all([this.Post.findOne({
-                where: {
-                  id: this.postA.get('id')
-                },
-                include: [
-                  { model: this.Tag, as: 'tags' },
-                  { model: this.Tag, as: 'categories' }
-                ]
-              }), this.Post.findOne({
-                where: {
-                  id: this.postB.get('id')
-                },
-                include: [
-                  { model: this.Tag, as: 'tags' },
-                  { model: this.Tag, as: 'categories' }
-                ]
-              }), this.Post.findOne({
-                where: {
-                  id: this.postC.get('id')
-                },
-                include: [
-                  { model: this.Tag, as: 'tags' },
-                  { model: this.Tag, as: 'categories' }
-                ]
-              })]);
-
-              expect(postA.get('categories').length).to.equal(1);
-              expect(postA.get('tags').length).to.equal(1);
-              expect(postB.get('categories').length).to.equal(1);
-              expect(postB.get('tags').length).to.equal(1);
-              expect(postC.get('categories').length).to.equal(1);
-              expect(postC.get('tags').length).to.equal(1);
-
-              expect(postA.get('categories')[0].get('type')).to.equal('category');
-              expect(postA.get('tags')[0].get('type')).to.equal('tag');
-              expect(postB.get('categories')[0].get('type')).to.equal('category');
-              expect(postB.get('tags')[0].get('type')).to.equal('tag');
-              expect(postC.get('categories')[0].get('type')).to.equal('category');
-              expect(postC.get('tags')[0].get('type')).to.equal('tag');
-            });
-          }
-        });
+          it('should create, find and include associations with scope values', async function() {
+            // We don't know the databaseVersion outside of the tests
+            const isMySQL8 = Support.getTestDialect() === 'mysql' && semver.satisfies(current.options.databaseVersion, '>=8.0.0');
```

Comment:
my only request would be to document why this test is disabled, for future maintainers/us :)

Reply:
That was mentioned later in the test I believe. I'll make an issue to debug it and re-enable the test later

Reply:
Oh yes you're right I remember seeing that last time